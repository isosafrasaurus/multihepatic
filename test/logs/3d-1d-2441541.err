INFO:    Using cached SIF image
INFO:    gocryptfs not found, will not be able to use gocryptfs
/usr/bin/python3: can't open file '/workspace/test/00_tacc_test.py': [Errno 2] No such file or directory
srun: error: c454-093: task 0: Exited with exit code 2
[JOB][FATAL] Command 'srun -n "${TASKS}" --export=ALL,LD_PRELOAD=,LD_AUDIT= "$APPTAINER_BIN" exec --cleanenv -B "$PWD:/workspace" "$IMAGE_URI" python3 "/workspace/$RUN_SCRIPT"' failed (rc=2)
