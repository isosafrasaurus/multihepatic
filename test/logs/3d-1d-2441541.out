[JOB] START 2025-10-06T14:01:30-05:00
[JOB] JobID: 2441541  Tasks: 1
[JOB] Script: test/00_tacc_test.py
[JOB] Fetching repository…
[JOB] Runtime: /opt/apps/tacc-apptainer/1.4.1/bin/apptainer (apptainer version 1.4.1-1.el9)
[JOB] Image  : docker://ghcr.io/isosafrasaurus/tacc-mvapich2.3-python3.12-graphnics:latest
[JOB] Running workload…
[JOB] ExitCode: 2
[JOB] END 2025-10-06T14:01:35-05:00
